stat/distmat/wishart.go

```diff
@@ -69,22 +69,18 @@ func NewWishart(v mat.Symmetric, nu float64, src rand.Source) (*Wishart, bool) {
 	return w, true
 }
 
-// MeanSym returns the mean matrix of the distribution as a symmetric matrix.
-// If x is nil, a new matrix is allocated and returned. If x is not nil, the
-// result is stored in-place into x and MeanSym will panic if the order of x
-// is not equal to the order of the receiver.
-func (w *Wishart) MeanSym(x *mat.SymDense) *mat.SymDense {
-	if x == nil {
-		x = mat.NewSymDense(w.dim, nil)
-	}
-	d := x.Symmetric()
-	if d != w.dim {
+// MeanSymTo calculates the mean matrix of the distribution in and stored it in dst.
```

Comment:
"... of the distribution and stores it in dst."?

Reply:
Thanks